src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+
+public class FigureSupplier {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureRandom = random.nextInt(5);
+        String color = ColorSupplier.getRandomColor();
```

Comment:
The 'ColorSupplier' instance 'colorSupplier' should be used here instead of calling a static method on 'ColorSupplier'. This will allow you to avoid using static methods in your solution.